torch/csrc/jit/codegen/cuda/scheduler/vectorize_helper.h

```diff
@@ -2,21 +2,15 @@
 
 #include <torch/csrc/jit/codegen/cuda/fusion.h>
 #include <torch/csrc/jit/codegen/cuda/ir_all_nodes.h>
-#include <torch/csrc/jit/codegen/cuda/iter_visitor.h>
 #include <torch/csrc/jit/codegen/cuda/scheduler/registry.h>
-#include <torch/csrc/jit/codegen/cuda/scheduler/utils.h>
+
+#include <vector>
 
 namespace torch {
 namespace jit {
 namespace fuser {
 namespace cuda {
-
-// TODO: Put implementations in a vectorize_helper.cpp
-namespace scheduler_utils {
-
-// Moved the definition of these to
-// torch/csrc/jit/codegen/cuda/scheduler/utils.cpp as making new CPP files is
-// painful for multiple reasons.
+namespace vectorize_helper {
```

Comment:
nit: Not a strong preference, but I'd like the same nesting relationship as the code directory, so in this case `scheduler::vectorizer_helper`.

Reply:
I think we should revisit our namespacing conventions after becoming a submodule. So for now, I don't really care which namespace we are putting our code into. Will merge this PR as is.